posts/tips-and-tricks-for-manually-scaling-a-global-dynamodb-table-from-an-aws-lambda.md

```diff
@@ -0,0 +1,128 @@
+---
+authors:
+- Dennis Sharpe
+tags:
+- AWS
+- Lambda
+- DynamoDB
+- Java
+date: 2020-11-16T15:52:00.000Z
+title: "Tips and Tricks for Manually Scaling a Global DynamoDB Table from an AWS Lambda"
+image:
+---
+# Objective
+Write an AWS Lambda that manually scales a global DynamoDB table
+
+# Why?
+DynamoDB tables can be set to automatically scale based on load. However, this process can take quite a while. We needed to scale up at a precise time to be ready for large batch loads.
```

Comment:
I would add some color to this.  What are some use cases where you would want to scale on load (ticketing or purchasing systems or specific events, ingestion of a large quantity of data (IoT?).
Also, why precise time? May want to make a counterpoint as to why we would want to do this right this instant as oppose to deferring to batch processing later.

Reply:
Expounding on this, it may be nice to show what a sample CW event would look like that could trigger this lambda based on a cron, or a sample SNS topic that triggers a lambda when a message is published to it etc.